extension/chrome/src/js/storage.js

```diff
@@ -167,12 +172,35 @@ const extensionStorage = (function() { // eslint-disable-line no-unused-vars
 				}
 			});
 	}
+
 	function saveTemplate(string) {
 		const data = {};
 		data[TEMPLATE_KEY] = string.split('\n');
 		return storagePromised.set(data);
 	}
 
+	function loadTemplateFromUrl(url) {
+		if (!url) {
+			return Promise.resolve()
+		}
+		return fetch(url)
+			.then(res => res.text())
```

Comment:
What happens when server responds with something else than 200? 401 Unauthorized will fail the promise?

Reply:
good point https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch
`The Promise returned from fetch() won’t reject on HTTP error status even if the response is an HTTP 404 or 500. Instead, it will resolve normally (with ok status set to false)`